### PR TITLE
Use sbt-dependency-graph 0.10.0-RC1

### PR DESCRIPTION
**Pull request by francisdb**

### CODE REVIEW

File: src/main/scala/com/lightbend/paradox/dependencies/DependenciesDirective.scala [hunk 1]
```diff
@@ -17,29 +17,29 @@
 package com.lightbend.paradox.dependencies
 
 import com.lightbend.paradox.markdown.LeafBlockDirective
-import net.virtualvoid.sbt.graph.{ModuleTree, ModuleTreeNode}
+import net.virtualvoid.sbt.graph.{Module, ModuleGraph} // .graph.{ModuleTree, ModuleTreeNode}
```
Comment (raboof) [inline src/main/scala/com/lightbend/paradox/dependencies/DependenciesDirective.scala]:
```suggestion
import net.virtualvoid.sbt.graph.{Module, ModuleGraph}
```

File: src/main/scala/com/lightbend/paradox/dependencies/ParadoxDependenciesPluginKeys.scala [hunk 2]
```diff
@@ -16,13 +16,12 @@
 
 package com.lightbend.paradox.dependencies
 
-import net.virtualvoid.sbt.graph.ModuleTree
-
+import net.virtualvoid.sbt.graph.{ModuleGraph}
```
Comment (raboof) [inline src/main/scala/com/lightbend/paradox/dependencies/ParadoxDependenciesPluginKeys.scala]:
```suggestion
import net.virtualvoid.sbt.graph.ModuleGraph
```

**Comment by francisdb** [comment]
@raboof cool these proposed edits, never seen that before

**Comment by francisdb** [comment]
do you plan to release this so we can fix https://github.com/akka/alpakka/issues/2145 ?

**Comment by ennru** [comment]
Yes, I'll try to make it happen.

**Pull request merged**